### PR TITLE
Botany flower additions

### PR DESCRIPTION
**Pull request by Pirill**
## About The Pull Request
Resprites lilies, geraniums and rainbow flowers away from being just poppy recolours, with new growth, item and wearable sprites as well as fitting descriptions

Adds forget-me-nots as a new flower mutation of the geranium, complete with its own set of sprites and info. It contains 20% kelotane as a reagent.

Adds wearable sprites for the rainbow flowers and fixes their reagent code (courtesy of @ivanmixo)

Adds craftable and wearable flower crowns from rainbow flowers (four variants!), sunflowers, poppies and lilies.

Lastly a number of small additions: Two new botany bounties for rainbow flower crowns and forget-me-nots, some small tweaks to botany bounty code, and a few typo and consistency fixes for a few other flowers.

unknown

Once more, special thanks to @ivanmixo @kit-katz and everyone else who gave feedback on these!

## Why It's Good For The Game

- Fewer copy pasted and recoloured sprites
- More content and cosmetics is good!

## Changelog
:cl:
imageadd: New sprites for geraniums, lilies and rainbow flowers
add: Forget-me-nots as a new flower type
add: Craftable flower crowns from rainbow flowers, sunflowers, poppies and lilies
add: Two new botany cargo bounties for forget-me-nots and rainbow flower crowns
fix: Fixes rainbow flower reagent code
/:cl:

### CODE REVIEW

**Comment by Pirill** [comment]
Oh and here's an image with all the new sprites if you wanna see that.
unknown (1)

File: code/modules/cargo/bounties/botany.dm [hunk 1]
```diff
@@ -3,12 +3,15 @@
 	var/datum/bounty/item/botany/multiplier = 0 //adds bonus reward money; increased for higher tier or rare mutations
 	var/datum/bounty/item/botany/bonus_desc //for adding extra flavor text to bounty descriptions
 	var/datum/bounty/item/botany/foodtype = "meal" //same here
+	var/datum/bounty/item/botany/format_exception = FALSE //Set to true if the bounty uses a custom format from the one below
```
Comment (Ivniinvi) [inline code/modules/cargo/bounties/botany.dm:6]: This shouldn't need this variable, just override the reward and desc on those specific bounties.

**Review by qwertyquerty — APPROVED** [review]

File: code/modules/cargo/bounties/botany.dm (hunk 1 above)
Comment (Ivniinvi) [reply code/modules/cargo/bounties/botany.dm:6]: Never mind I guess

Comment (Pirill) [reply code/modules/cargo/bounties/botany.dm:6]: Yep that was the first thing I tried, but no dice. The New() code overwrote anything I did.

**Comment by kit-katz** [comment]
fastest merge in the west

**Pull request merged**